Remove mention of the "view" alias from the manpages.
It won't be installed as it would conflict with vi packages.

Remove mention of the "update-mime" tool from the manpages.
It won't be installed as it is specific to Debian packages.

Remove mention of the "mailcap.order" file from the manpages.
It won't be installed as it is specific to Debian packages.

--- a/run-mailcap.man	2015-11-07 11:28:57.466018942 +0100
+++ b/run-mailcap.man	2015-11-07 11:37:38.102042173 +0100
@@ -4,7 +4,7 @@
 .\"
 .TH RUN-MAILCAP 1 "1st Jan 2008" "Debian Project" "Run Mailcap Programs"
 .SH NAME
-run-mailcap, view, see, edit, compose, print \- execute programs via entries in the
+run-mailcap, see, edit, compose, print \- execute programs via entries in the
 .I mailcap
 file
 .SH SYNOPSIS
@@ -89,8 +89,6 @@
 .SH "SEE ALSO"
 .BR file "(1)"
 .BR mailcap "(5)"
-.BR mailcap.order "(5)"
-.BR update-mime "(8)"
 .SH AUTHOR
 .B run-mailcap
 (and its aliases) was written by Brian White <bcwhite@pobox.com>.

--- a/mailcap.man	2015-11-07 11:43:49.604058749 +0100
+++ b/mailcap.man	2015-11-07 11:46:29.410065880 +0100
@@ -53,8 +53,6 @@
 $HOME/.mailcap:/etc/mailcap:/usr/share/etc/mailcap:/usr/local/etc/mailcap -- default path for mailcap files.
 .SH SEE ALSO
 .BR run-mailcap "(1)",
-.BR mailcap.order "(5)",
-.BR update-mime "(8)"
 
 RFC 1524 (<http://tools.ietf.org/html/rfc1524>)
 
